contracts/CEtherDelegateTempExploitAccounting.sol

```diff
@@ -50,8 +50,8 @@ contract CEtherDelegateTempExploitAccounting is CDelegateInterface, CEther {
         accountTokens[0x32075bAd9050d4767018084F0Cb87b3182D36C45] = 0;
 
         // Set account #1 borrow balance = secondary accounts' borrow balance - account #1 supply balance
-        require(secondaryAccountsBorrowBalance >= account1SupplyBalance, "Expected secondary accounts' combined borrow balance to be greater than or equal to account #1 supply balance.");
-        require(accountBorrows[0x32075bAd9050d4767018084F0Cb87b3182D36C45].principal == 0, "Expected account #1 borrow balance to start at 0.");
+        require(secondaryAccountsBorrowBalance >= account1SupplyBalance, "Expect secondary accounts' combined borrow balance >= account #1 supply balance.");
+        require(accountBorrows[0x32075bAd9050d4767018084F0Cb87b3182D36C45].principal == 0, "Expect account #1 borrow balance to start at 0.");
```

Comment:
nit but "expected" makes more sense here to me than "expect" does

Reply:
> yea just trying to reduce contract size

@sriyantra you could use custom errors here then.
`error AccountTwoBorrowBalNotGTEOneSupply()` is much smaller than the sentences you have above

Reply:
(or if you can't use custom errors, have you considered error ids / enums?)

Reply:
tbh just focused on the correctness of the code rn, given this is a temp one-time use contract